packages/tokamak/sdk/src/adapters/native-token-bridge.ts

```diff
@@ -80,15 +45,9 @@ export class NativeTokenBridgeAdapter implements IBridgeAdapter {
     )
 
     return events
-      .filter((event) => {
-        // Specifically filter out ETH. ETH deposits and withdrawals are handled by the ETH bridge
-        // adapter. Bridges that are not the ETH bridge should not be able to handle or even
-        // present ETH deposits or withdrawals.
-        return (
-          !hexStringEquals(event.args.l1Token, ethers.constants.AddressZero) &&
-          !hexStringEquals(event.args.l2Token, predeploys.OVM_ETH)
-        )
-      })
+      .filter((event) =>
+        this.supportsTokenPair(event.args.l1Token, event.args.l2Token)
```

Comment:
Thank you very much!

Then, do we get all of ERC20DepostiInitiated not only NativeToken? 🙇🏼‍♂️

Reply:
I think we can get only for
ERC20DepositInitiated(nativeTokenAddress, Predeploys.LEGACY_ERC20_NATIVE_TOKEN, address, ...)

Reply:
nativeTokenAddress
We can get above value using L1StandardBridge.nativeTokenAddress() 🙇🏼‍♂️